src/pages/ScholarshipForm.jsx

```diff
@@ -48,9 +62,20 @@ function ScholarshipForm() {
         />
         {requiredTextField('description', 'Description')}
         {requiredTextField('website', 'Website')}
-        <Button variant="contained" color="primary">
-          Submit
-        </Button>
+        <ScholarshipAmountField
+          amountType={formFieldStates.amountType}
+          minAmount={formFieldStates.minAmount}
+          maxAmount={formFieldStates.maxAmount}
+          onTypeChange={updateFn('amountType')}
+          onMinChange={updateAmount('minAmount')}
+          onMaxChange={updateAmount('maxAmount')}
+          updateMinMax={updateMinMax}
```

Comment:
You could instead provide a single `updateMinMax` function that accepts a min and max as parameters. The other functions could be derived from it